sale_order_line_price_history/__manifest__.py

```diff
@@ -2,14 +2,15 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Sale order line price history",
-    "version": "11.0.1.0.0",
+    "version": "12.0.1.0.0",
     "category": "Sales Management",
     "author": "Tecnativa, "
               "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/sale-workflow/",
+    "summary": "Sale order line price history",
     "license": "AGPL-3",
     "depends": [
-        "sale",
+        "sale_management",
```

Comment:
Why is this needed?

Reply:
@pedrobaeza To display the Main Sales menu.

Reply:
@murtuzasaleh  Have a look at https://github.com/OCA/sale-workflow/pull/606#issuecomment-364018576